### DISCUSSION TITLE
Problems

### DISCUSSION THREAD

**Discussion by Ruben-Schadron** [community/community#40060] [Codespaces]
### Select Topic Area

Bug

### Body

I can't start my bot on codespace 
![image](https://user-images.githubusercontent.com/77514111/203946947-6534519a-59ba-4eaf-910d-bd2bc3109d4c.png)
it is just for testing that i want to use it. Is there a way that i can fix that. I tried to remove the node_module folder and reinstalled every package but it still don't work

**Comment by DSDmark** [comment]
Hi @Ruben-Schadron,

can you give link of that repo.

**Reply by Ruben-Schadron** [reply]
@DSDmark i added you to the repo and there is also the link https://github.com/Ruben-Schadron/spooky-djs14

**Reply by DSDmark** [reply]
Hi @Ruben-Schadron,

when I test your **bot** project. I have seen some incompatibly errors. 

![2022-11-25_19-18](https://user-images.githubusercontent.com/85744816/203997057-d64e00ac-c2b0-438f-955f-e8eff9a85a32.png)

I think it is your **node** engine issue. you node **version** not understanding that ```??=``` syntax. I guess, update your node version. Then try.

Why the way, I'm using node 14 version that why. I also not updated 🥲.

**Reply by Ruben-Schadron** [reply]
@DSDmark do you know if there is a way to update it

**Reply by DSDmark** [reply]
@Ruben-Schadron, which os you are using.

**Reply by Ruben-Schadron** [reply]
@DSDmark it is just code space on github

**Comment by DSDmark** [comment]
Hi @Ruben-Schadron,

first, sorry for not response you, I have a lot of work in these days. So,

So, About updating your codespace **node version**. There are several methods, you're used to configure your codespace.

1. open your **codespace** and pass ```ctrl+p``` for **Command Palette**.

![png](https://user-images.githubusercontent.com/85744816/204007482-8a97484c-7e0e-47f3-af12-06d5aa21be10.png)

3. search ***dev container configuration*** and click and search ***node*** and select your version.

![2022-11-25_20-39](https://user-images.githubusercontent.com/85744816/204007517-cb8a607c-c93d-4152-a7b9-507ddb5151ab.png)

4. after that, search ***node*** again and select.

![2022-11-25_20-41](https://user-images.githubusercontent.com/85744816/204008049-3c4edbcb-2839-4635-9b3e-6a5d7e47f4d8.png)

6. in the last open again ***Command Palette*** and type ```>rebuild```.

Check your node version after your codespace run. 
```node -v```

**Reply by Ruben-Schadron** [reply]
@DSDmark thanks

you can now also remove the repo from your account
